ner_v2/detectors/numeral/number_range/standard_number_range_detector.py

```diff
@@ -179,8 +179,33 @@ def detect_number_range(self, text):
         for detector in self.detector_preferences:
             number_list, original_list = detector(number_list, original_list)
             self._update_tagged_text(original_list)
+        number_list, original_list = self._add_absolute_numbers(number_list, original_list)
         return number_list, original_list
 
+    def _add_absolute_numbers(self, number_list, original_list):
+        number_abs_list = number_list or []
+        original_abs_list = original_list or []
+        abs_number_pattern = re.compile(ur'({number}\d+)'.format(number=numeral_constant.NUMBER_REPLACE_TEXT),
+                                        re.UNICODE)
+        abs_number_matches = abs_number_pattern.findall(self.processed_text)
+        for match in abs_number_matches:
+            if self.unit_type:
+                if self.number_detected_map[match].entity_value[numeral_constant.NUMBER_DETECTION_RETURN_DICT_UNIT] \
+                        is not None:
+                    number_abs_list.append({numeral_constant.NUMBER_RANGE_MAX_VALUE: None,
+                                            numeral_constant.NUMBER_RANGE_MIN_VALUE: None,
+                                            numeral_constant.NUMBER_RANGE_VALUE_UNIT: self.unit_type,
```

Comment:
correct unit type, it should be equal to `self.number_detected_map[match].entity_value[numeral_constant.NUMBER_DETECTION_RETURN_DICT_UNIT]`